crates/burn-core/src/nn/tanh.rs

```diff
@@ -0,0 +1,27 @@
+use crate as burn;
+
+use crate::module::Module;
+use crate::tensor::backend::Backend;
+use crate::tensor::Tensor;
+
+/// Applies the rectified linear unit function element-wise
+/// See also [tanh](burn::tensor::activation::tanh)
+///
```

Comment:
The comments on not correct, also shouldn't have an empty comment line before the struct.